src/main/java/it/pagopa/pn/commons/log/PnAuditLog.java

```diff
@@ -0,0 +1,22 @@
+package it.pagopa.pn.commons.log;
+
+import ch.qos.logback.classic.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PnAuditLog {
+
+    private PnAuditLog() {
+        throw new UnsupportedOperationException();
+    }
+
+    static final Logger logger = (Logger) LoggerFactory.getLogger(PnAuditLog.class);
```

Comment:
https://stackoverflow.com/questions/8297705/how-to-implement-thread-safe-lazy-initialization

E' possibile fare lazy loading utilizzando inner class

Reply:
Vantaggi? Minor tempo di startup?